security/compliance_operator/compliance-operator-release-notes.adoc

```diff
@@ -15,6 +15,52 @@ For an overview of the Compliance Operator, see xref:../../security/compliance_o
 
 To access the latest release, see xref:../../security/compliance_operator/compliance-operator-updating.adoc#olm-preparing-upgrade_compliance-operator-updating[Updating the Compliance Operator].
 
+[id="compliance-operator-release-notes-1-1-0"]
+== OpenShift Compliance Operator 1.1.0
+
+The following advisory is available for the OpenShift Compliance Operator 1.1.0:
+
+* link:https://access.redhat.com/errata/RHBA-2023:3630[RHBA-2023:3630 - OpenShift Compliance Operator bug fix and enhancement update]
+
+[id="compliance-operator-1-1-0-new-features-and-enhancements"]
+=== New features and enhancements
+
+* A start and end timestamp is now available in the `ComplianceScan` CRD status.
+
+* The Compliance Operator can now be deployed on Hosted Control Planes using the OperatorHub by creating a Subscription file. Fore more information, see xref:../../security/compliance_operator/compliance-operator-installation.html#installing-compliance-operator-hcp_compliance-operator-installation[Installing the Compliance Operator on Hosted Control Planes].
```

Comment:
Noticed this in an earlier file too - "Subscription file". Should that be one of:

```
subscription file
```
or
```
`Subscription` file
```